src/intellisense/serviceRegistry.web.ts

```diff
@@ -15,12 +13,6 @@ export function registerTypes(serviceManager: IServiceManager, _isDevMode: boole
         IExtensionSyncActivationService,
         NotebookCellBangInstallDiagnosticsProvider
     );
-    serviceManager.addSingleton<NotebookCellLanguageService>(NotebookCellLanguageService, NotebookCellLanguageService);
```

Comment:
Cell Language service is responsible for checking the language type for a cell, which is different from autocomplete and other rich language features, thus moving it to `src/notebooks`.

It seems that `src/intellisense` component would need to have the knowledge of notebooks/iw/kernels, so it's likely not going to be a shared component.